### ISSUE TITLE
What am I missing?

### ISSUE THREAD

**Issue by probal** [jeffwillette/react-awesome-resume#1]
```
import React from 'react';
import ReactDOM from 'react-dom';
import registerServiceWorker from './registerServiceWorker';

import Resume from 'react-awesome-resume';
import 'semantic-ui-css/semantic.min.css';

const  myJSONResume = require('./person.json')

ReactDOM.render(
    <Resume person={myJSONResume} theme="default" />,
    document.getElementById("root")
  );
registerServiceWorker();

```
This doesn't render data from my person.json, rather it renders the default person data, that is Erlich Bachman. Could you please point, what am I missing here? person.json sample is taken from jsonresume.org

**Comment by jeffwillette** [comment]
can you try printing your person to the console after the import to make sure that it has actually been imported? It only uses the stock "person" as a default prop, so only when the person passed as a prop is empty...

https://github.com/deltaskelta/react-awesome-resume/blob/master/src/index.js#L7-L20

**Comment by probal** [comment]
Thanks for your response. Here is my code snippet and output screenshot for your perusal.

```
import React from 'react';
import ReactDOM from 'react-dom';
import './index.css';
import registerServiceWorker from './registerServiceWorker';
import Resume from 'react-awesome-resume';
import 'semantic-ui-css/semantic.min.css';
const  myJSONResume = require('./person.json')
console.log('myJSONResume: ')
console.log(myJSONResume)
ReactDOM.render(
    <Resume person={myJSONResume} theme={"default"} inline={false} />,
    document.getElementById("root")
  );
registerServiceWorker();
```

Console Log: 
<img width="1203" alt="screen shot 2017-12-26 at 9 52 10 am" src="https://user-images.githubusercontent.com/1135164/34346536-c6e5ee3c-ea22-11e7-81ac-4070cbf96471.png">

**Comment by jeffwillette** [comment]
How are you running react? is it from `create-react-app` or a custom setup, or something else? can you give me exact steps to reproduce?

**Comment by probal** [comment]
Steps to reproduce:

1.  create-react-app myapp
2.  cd myapp
3.  npm install react-awesome-resume semantic-ui-css
4.  Then create a person.json in src folder, copy default schema from 
      https://jsonresume.org/schema/
5.  Change index.js to 

```
import React from 'react';
import ReactDOM from 'react-dom';
import Resume from 'react-awesome-resume'
import 'semantic-ui-css/semantic.min.css';
import registerServiceWorker from './registerServiceWorker';

const  myJSONResume = require('./person.json')
console.log('myJSONResume: ')
console.log(myJSONResume)

ReactDOM.render(
    <Resume person={myJSONResume} theme={"default"} inline={false} />,
    document.getElementById("root")
  );
registerServiceWorker();
```

6.  npm start
7. Page loads with stock default person data rather than my custom person.json, but console log shows my custom person.json, as you see in the screenshot.

**Comment by jeffwillette** [comment]
oh I see now, I goofed the prop name in the readme... try changing to this...

```
<Resume jsonResume={myJSONResume} theme={"default"} inline={false} />
```

**Comment by probal** [comment]
Yeah right :)

I missed the prop name 

const Resume = ({ theme, **jsonResume**, inline }) => {
  const ThemedResume = themes[theme];
  return <ThemedResume inline={inline} jsonResume={jsonResume} />;
};

Thanks for your help 👍

**Comment by jeffwillette** [comment]
Fixed README in 0e75900
